.github/workflows/tests.yaml

```diff
@@ -5,6 +5,14 @@ on: [push, pull_request]
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    env:
+      CHECKPOINT_DIR: /var/lib/kubelet/checkpoints
+      REGISTRY_PORT: 5000
```

Comment:
Would it be possible to move the changes that add local registry and build/push the image and in a separate commit?

Reply:
Yea, actually I was doin it.